### PR TITLE
Responsive

### PR DESCRIPTION
**Pull request by Juliocho990**
### Summary
- 1024px breakpoint was used as media query
- CSS book detail properties were used to create a responsive card.

### Screenshot
book detail responsive

### Trello card
https://trello.com/c/UueZQ0IJ/34-responsive

### CODE REVIEW

File: scss/_mixins.scss [hunk 1]
```diff
@@ -0,0 +1,5 @@
+@mixin for-mobile {
+  @media (max-width: 1024px) {
+    @content;
+  }
+}
```
Comment (CrisVega08) [inline scss/_mixins.scss]: Add empty line

File: scss/_variables.scss [hunk 2]
```diff
@@ -0,0 +1,2 @@
+$badge-heigth: 120px;
+$badge-width: 94px;
```
Comment (CrisVega08) [inline scss/_variables.scss]: Same

File: scss/_mixins.scss [hunk 3]
```diff
@@ -0,0 +1,5 @@
+@mixin for-mobile {
+  @media (max-width: 1024px) {
```
Comment (CrisVega08) [inline scss/_mixins.scss]: You can use 1024 like a variable

**Review by CrisVega08 — CHANGES_REQUESTED** [review]

File: scss/_mixins.scss (hunk 3 above)
Comment (Juliocho990) [reply scss/_mixins.scss]: ok

File: scss/_variables.scss (hunk 2 above)
Comment (Juliocho990) [reply scss/_variables.scss]: ok

File: scss/_mixins.scss (hunk 1 above)
Comment (Juliocho990) [reply scss/_mixins.scss]: ok

**Review by eledwinn — APPROVED** [review]

**Pull request merged**